src/localforage.js

```diff
@@ -118,50 +118,58 @@ class LocalForage {
         // If the options argument is an object, we use it to set values.
         // Otherwise, we return either a specified config value or all
         // config values.
-        if (typeof options === 'object') {
-            // If localforage is ready and fully initialized, we can't set
-            // any new configuration values. Instead, we return an error.
-            if (this._ready) {
-                return new Error(
-                    "Can't call config() after localforage " + 'has been used.'
-                );
-            }
-
-            for (let i in options) {
-                if (i === 'storeName') {
-                    options[i] = options[i].replace(/\W/g, '_');
+        try {
```

Comment:
I don't think this `try/catch` block here is helpful to add, as it will prevent unexpected errors from throwing.